include/FakeSetupManager.h

```diff
@@ -0,0 +1,14 @@
+#ifndef FAKESETUPMANAGER_H
```

Comment:
Not very important but you can also use the `#pragma once` syntax (to stay consistent with the other files)